src/strings.cpp

```diff
@@ -515,6 +515,12 @@ static char *FormatGenericCurrency(char *buff, const CurrencySpec *spec, Money n
 	return buff;
 }
 
+static char *FormatModifierKey(char *buff, ModifierKey key, const char *last) {
+	int64 args[] = { 1 };
+	StringParameters tmp_params(args);
```

Comment:
`StringParameters tmp_params(nullptr, 0, nullptr);`

Reply:
`error: no matching function for call to ‘StringParameters::StringParameters(std::nullptr_t, int, std::nullptr_t)`
I guess something changed since so I just use `MakeParameters()`, that seems to work.